src/Structs/Permissions.js

```diff
@@ -9,16 +9,31 @@ class Permissions {
      */
     permissions = 0
 
-    constructor (bit) {
-        this.permissions = bit
+    constructor (bitfield) {
+        if (Array.isArray(bitfield))
+            this.set(bitfield)
+        else if (typeof bitfield === "number")
+            this.permissions = bitfield
+    }
```

Comment:
Make the isArray a single line "if this then nothing else"-type of method with the else-if being on it's own at the bottom.